src/adapter/src/coord/sequencer/inner/create_index.rs

```diff
@@ -228,6 +228,7 @@ impl Coordinator {
             self.allocate_transient_id()?
         };
         let optimizer_config = optimize::OptimizerConfig::from(self.catalog().system_config())
+            .override_from(&self.catalog.get_cluster(*cluster_id).config.features())
```

Comment:
Should we wire this up also in `subscribe.rs`? (maybe also copy_to?)

Reply:
Added to `subscribe.rs`. The `COPY TO` pipeline piggy-backs on `inner/peek.rs` so it's already handled.